src/ts/components/Match.tsx

```diff
@@ -34,10 +37,26 @@ class Match<TMatch extends IMatch> extends Component<IMatchProps<TMatch>> {
               />
             </div>
           )}
+          {this.props.feedbackHref && (
+            <div className="MatchWidget__feedbackLink">
+              <a
+                target="_blank"
+                href={this.getFeedbackLink(feedbackInfo)}
+              >
+                Something's not right? Tell us!
+              </a>
+            </div>
+          )}
         </div>
       </div>
     );
   }
+
+  private getFeedbackLink = (feedbackInfo: any) => {
+    const data = encodeURIComponent(JSON.stringify(feedbackInfo, undefined, 2))
+    return this.props.feedbackHref! + data
```

Comment:
Could we pass `feedbackHref` as an argument to this function, rather than referencing the prop? It seems like we'd only want to call this when we definitely have the href.

Reply:
Done.  Although the logic made that implicit (only called when feedbackHref exists) that makes it explicit.